wasm2c/wasm-rt.h

```diff
@@ -56,35 +52,53 @@ extern "C" {
  * This is usually 10%-25% faster, but requires OS-specific support.
  */
 
-/** Check whether the signal handler is supported at all. */
-#if (defined(__linux__) || defined(__unix__) || defined(__APPLE__)) && \
-    defined(__WORDSIZE) && __WORDSIZE == 64
```

Comment:
Why not keep this check as is?

Reply:
The `(defined(__linux__) || defined(__unix__) || defined(__APPLE__))` is no longer needed, as we support both POSIX and windows. The `defined(__WORDSIZE) && __WORDSIZE == 64` approach to testing 64-bit architectures does not work on msvc (https://godbolt.org/z/af3E66s6G)